data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -1,8 +1,10 @@
+import enum
+import pathlib
```

Comment:
a real nit -- since we import pathlib, can we refer to Path as pathlib.Path and then delete line 4? (I am sorry)

Reply:
Yessss eagle eyes! I love it. 

TBH I'm always in favor of importing top-level modules and not specific items from the module. It is a little chattier, but the code is more readable because you know where that thing came from.